owlbot.py

```diff
@@ -22,5 +22,5 @@
     '.kokoro/dependencies.sh',
     'codecov.yaml',
     'renovate.json',
-    '.kokoro/build.sh',
+    '.kokoro/build.sh'
```

Comment:
This should be removed, right?

Reply:
I had to bring this back in to exclude the arrow tests in Java 17 until https://github.com/googleapis/java-bigquery/issues/2060 is resolved.